Update the version Control section with the latest changes (#861)
codenvy/che-docs: src/main/pages/che-7/end-user-guide/proc_accessing-a-git-repository-via-ssh.adoc

.Prerequisites
* Personal link:https://help.github.com/en/articles/types-of-github-accounts[GitHub account] or other Git provider account created.

== Generating an SSH key

> A common SSH key, that will work for all Git providers, is already present by default. You just need to add the public key to the Git provider.

Reviewer: ```suggestion
A common SSH key that works with all Git providers is present by default. To start using it, add the public key to the Git provider.
```

> To generate an SSH key pair that will work only with a particular Git provider:

Reviewer: ```suggestion
To generate an SSH key pair that only works with a particular Git provider:
```

> . Run the *SSH: generate key pair for particular host*.

Reviewer: ```suggestion
. Run the *SSH: generate key pair for particular host* command.
```
